config/crd/bases/submariner.io_submariners.yaml

```diff
@@ -71,6 +71,25 @@ spec:
                 type: boolean
               globalCIDR:
                 type: string
+              healthCheckSpec:
+                properties:
+                  enabled:
+                    type: boolean
+                  intervalSeconds:
+                    description: The interval upon which HealthChecker ping packets
+                      are sent
```

Comment:
```suggestion
                    description: The interval at which health check pings are sent
```

Reply:
This is a  generated file , the API update should resolve this